fastlane/Fastfile

```diff
@@ -74,13 +76,13 @@ lane :update_native_dependencies_to_latest do |options|
 end
 
 desc "Update purchases-ios dependency version"
-lane :update_ios_native_version do |options|
+private_lane :update_ios_native_version do |options|
   new_version_number = options[:new_version]
   previous_version_number = options[:current_version]
   files_to_update = [
-    'PurchasesHybridCommon.podspec',
-    'ios/PurchasesHybridCommon/Podfile',
-    'DEVELOPMENT.md',
+    File.join(get_root_folder, 'PurchasesHybridCommon.podspec'),
+    File.join(get_root_folder, 'ios/PurchasesHybridCommon/Podfile'),
+    File.join(get_root_folder, 'DEVELOPMENT.md')
```

Comment:
So I changed this because I was finding some weird behavior with fastlane when replacing the versions. 

Basically we were doing `Dir.chdir(get_root_folder)` during the `update_native_version ` lane. However, after calling the `replace_text_in_files` action from the plugin, the path changed to the parent of the project path. This did not happen if we did not call `Dir.chdir(get_root_folder)`. In order to solve this, while keeping the ability to run this from anywhere, I changed it to passing absolute paths like this. That said I do wonder if we want to support running this lane from different directories, since in most cases it will be run from the root folder and this wouldn't be needed and we can just use relative paths.

Reply:
I think we can just keep it simple and use relative paths like we do in iOS